etc/docker/demo/setup_data.py

```diff
@@ -1,24 +1,75 @@
-#/usr/local/bin python
-# Copyright 2014-2018 CERN for the benefit of the ATLAS collaboration.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# You may not use this file except in compliance with the License.
-# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
-#
-# Authors:
-# - Thomas Beermann, <thomas.beermann@cern.ch>, 2018
+#!/usr/local/bin python
+"""
```

Comment:
Are we going for """ or # now? @vingar went for # in his PR. I feel like we are moving in circles there 😆

Reply:
I'm strongly opinionated towards #

Reply:
and in any case hardcoding /usr/local/bin/python looks weird, shouldn't this be /usr/bin/env python ?

Reply:
I'm also in favour of #. I just unitized all headers for the flask files to use #.

Reply:
# is the recommended way. there is an issue about it #??. I'll submit the script to auto generate the header (git log etc) tomorrow. It' helps a lot

Reply:
:man_facepalming: 
I'm lost I thought we decided the """

Reply:
Let it be decided for all eternity that we use # as the header character.

Reply:
see #534